orachk.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# change these as needed
+# will also work with exachk
+
+export PATH=/usr/local/bin:$PATH
+export ORAENV_ASK=NO
+. /usr/local/bin/oraenv 
```

Comment:
If ORACLE_SID isn't set or isn't in oratab, I think this command waits on a prompt, which can be an issue if run non-interactively.  I like to add a "< /dev/null" to avoid hangs in this case.  May need a "set -e" first for this case to exit the script itself.

Reply:
Actually when I think about it, unless the user is quite familiar with Oracle, requiring a certain environment variable to be set isn't super intuitive.  What do you think about taking the SID as an optional argument?

Reply:
Yes, I can do that.